R/nested.R

```diff
@@ -24,12 +24,13 @@
 #'     - Function calls
 #'     - Function definitions
 #' - Remove `includeText = TRUE`
+#' @importFrom purrr rerun
 compute_parse_data_nested <- function(text) {
   parsed <- parse(text = text, keep.source = TRUE)
   parse_data <- tbl_df(utils::getParseData(parsed, includeText = TRUE))
   pd_nested <-
     parse_data %>%
-    mutate_(child = ~rep(list(NULL), length(text))) %>%
+    mutate_(child = ~rerun(length(text), tibble())) %>%
```

Comment:
Ok. Yeah I was thinking about modifying `bind_rows()` too. I just thought it might add extra complication for little gain. 
Anyways, maybe we can even prevent the tibble from being created.
```r
bind_not_null <- function(...) {
  if (is.null(unlist(...))) return(NULL)
  bind_rows(...)
}
```
I can branch out and adapt the solution presented in this PR to match the one outlined above and benchmark the two in terms of speed (which is your ultimate concern I guess, not memory itself) so we can make an informed decision if that's ok.